test/cassandro_soft_delete_test.rb

```diff
@@ -39,6 +38,62 @@ class Admin < Cassandro::Model
     assert !Admin.all.include?(me)
 
   end
+
+  test "#where not includes deleted" do
+    me = Admin.create(:nickname => "tarolandia")
+    other = Admin.create(:nickname => "Jim")
+
+    me.destroy
+
+    assert_equal 0, Admin.where(:nickname, "tarolandia").size
+  end
+
+  test "#where should include deleted if asked" do
+    me = Admin.create(:nickname => "tarolandia")
+    other = Admin.create(:nickname => "Jim")
+
+    me.destroy
+
+    assert_equal 1, Admin.where(:nickname, "tarolandia", true).size
+  end
+
+  test "#count not includes deleted" do
+    me = Admin.create(:nickname => "tarolandia")
+    other = Admin.create(:nickname => "Jim")
+
+    me.destroy
+
+    assert_equal 1, Admin.count
+    assert_equal 0, Admin.count(:nickname, "tarolandia")
+  end
+
+  test "#count should include deleted if asked" do
+    me = Admin.create(:nickname => "tarolandia")
+    other = Admin.create(:nickname => "Jim")
+
+    me.destroy
+
+    assert_equal 2, Admin.count(true)
+    assert_equal 1, Admin.count(:nickname, "tarolandia", true)
```

Comment:
I'm not sure about this assertion. I would test it like:

```Ruby
assert_equal 1, Admin.count
assert_equal 2, Admin.count(true)
```
Since what you're trying to prove is that `count` with `true` includes the deleted records.

Thoughts?